### DISCUSSION TITLE
Huron County Michigan

### DISCUSSION THREAD

**Discussion by fhangrin** [KillBait/FS19_AutoDrive_MapImages#31] [General]
Any chance of adding Huron County Michigan (X16)?

**Comment by KillBait** [comment]
Thanks for the suggestion, this has been added.

Just a quick note, looks like the map author did'nt bother to mark the fields
on the PDA image and replied on FS22 adding it, it looks very wasted out.

I'm working on a different image, but its a lot of work in GIMP to get it even
usable.
